### DISCUSSION TITLE
Vercel doesn't find my github repo for deployment

### DISCUSSION THREAD

**Discussion by rebeccapeltz** [vercel/community#43] [Help] [answered]
It's public: https://github.com/rebeccapeltz/test-u
Any one have suggestions on why it can't find it.  It gives me a toast message when I press deploy.

**Comment by rebeccapeltz** [comment, marked as answer]
I figured it out - I needed to go to github settings and make this repo accessbile to Vercel in settings | applications.

**Reply by mcsdevv** [reply]
Hey @rebeccapeltz, I'm glad you got this sorted, we're working on an update that will make this much clearer - it should be available soon!

**Reply by Milky-Egypt501** [reply]
I have the same issue and I can't find my way around. I went to settings, applications, but I don't know what to do after that

**Reply by HPouyanmehr** [reply]
> I have the same issue and I can't find my way around. I went to settings, applications, but I don't know what to do after that

Hi
You can try this:
1. Go to your GitHub settings
2. In the Applications settings you should see Vercel app in the "Installed GitHub Apps" tab.*
3. Click on configure button and in the "Repository access" section you can configure the access as you like.

*In case you cannot see Vercel app in your installed ones first go to [this page](https://github.com/apps/vercel) and install the app by clicking on the install button, then come back and follow the second and third steps above.

**Reply by PaulHaze** [reply]
@hajhosein Thanks for this, I swear the layout has changed recently as I used to have no problems with deployment but have just spent the last hour googling and googling about how to add vercel to my repo! There definitely seems to be a disconnect in the Vercel docs to how things are laid out now. Your solution fixed it for me though :)

**Reply by aminsoraya** [reply]
great

**Reply by amandaalexandre** [reply]
> > I have the same issue and I can't find my way around. I went to settings, applications, but I don't know what to do after that
> 
> Hi You can try this:
> 
> 1. Go to your GitHub settings
> 2. In the Applications settings you should see Vercel app in the "Installed GitHub Apps" tab.*
> 3. Click on configure button and in the "Repository access" section you can configure the access as you like.
> 
> *In case you cannot see Vercel app in your installed ones first go to [this page](https://github.com/apps/vercel) and install the app by clicking on the install button, then come back and follow the second and third steps above.

Unfortunately, GitHub is not loading this page.
![image](https://user-images.githubusercontent.com/15349795/194294113-e36e542b-c5ba-42c8-a4cf-533c868e70aa.png)

**Reply by HPouyanmehr** [reply]
> > > I have the same issue and I can't find my way around. I went to settings, applications, but I don't know what to do after that
> > 
> > 
> > Hi You can try this:
> > 
> > 1. Go to your GitHub settings
> > 2. In the Applications settings you should see Vercel app in the "Installed GitHub Apps" tab.*
> > 3. Click on configure button and in the "Repository access" section you can configure the access as you like.
> > 
> > *In case you cannot see Vercel app in your installed ones first go to [this page](https://github.com/apps/vercel) and install the app by clicking on the install button, then come back and follow the second and third steps above.
> 
> Unfortunately, GitHub is not loading this page. ![image](https://user-images.githubusercontent.com/15349795/194294113-e36e542b-c5ba-42c8-a4cf-533c868e70aa.png)

I've checked the link and it's ok. https://github.com/apps/vercel

**Reply by amyegan** [reply]
Thanks for reaching out @amandaalexandre! There was [a GitHub incident](https://www.githubstatus.com/incidents/gq1x0j8bv67v) a few hours ago that could have affected the page. Are you still having trouble accessing [github.com/apps/vercel](https://github.com/apps/vercel)? If so, are there any errors you can see printed in your browser console or network log?

**Reply by Scott2bReal** [reply]
I'm trying to install the vercel github app and landing on the same page as @amandaalexandre with a 502 status code. I couldn't find any errors in the console or network tab.

**Reply by SouthCloud0703** [reply]
@Scott2bReal Me too

**Reply by Scott2bReal** [reply]
Hey @YutaroNagumo! I submitted a bug report to GitHub about this, and they got back to me saying the issue was related to me being a collaborator in an org with many repositories. I removed myself from that org and was able to get Vercel up and running :)

**Reply by didierfranc** [reply]
Same issue here "This page is taking too long to load" when I'm logged in on Github. That will be hard to remove me from my own org ...

**Reply by Aizen-sama214** [reply]
Hi, is there any workaround for this which doesn't require me to remove myself from the org ? Not able to get Vercel for my personal account.

**Comment by amandaalexandre** [comment]
Still shows the same error for me. I'll wait more hours, I guess

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by amyegan** [reply]
I don't see any active incidents at GitHub or Vercel at the moment. The page should be available. Are you seeing any additional error information in your browser dev tools?

**Comment by amandaalexandre** [comment]
Yes, I get a 502 error.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by amyegan** [reply]
Oh no! I think you should [submit a bug report](https://support.github.com/contact/bug-report) to GitHub to let them know exactly what is happening

**Comment by amandaalexandre** [comment]
Thanks for your help. I'm getting the same error on Netlify, so I guess the problem is on my end.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Scott2bReal** [reply]
Hey @amandaalexandre! You may have already worked this out, but I figured I'd chime in. I submitted a bug report to GitHub, and they got back to me saying the issue was related to me being a collaborator in an org with many repositories. I removed myself from that org and was able to get Vercel up and running.

**Comment by amandaalexandre** [comment]
I'm not able to remove myself from said organization at this point. I don't know what I can do.

Amanda Alexandre

________________________________

**Comment by Aizen-sama214** [comment]
> Hey @YutaroNagumo! I submitted a bug report to GitHub about this, and they got back to me saying the issue was related to me being a collaborator in an org with many repositories. I removed myself from that org and was able to get Vercel up and running :)

Hi @Scott2bReal , is there any workaround for this which doesn't require me to remove myself from the org ? Not able to get Vercel for my personal account.

**Reply by Scott2bReal** [reply]
Sorry, I wound up removing myself from the org to fix it, so I didn't investigate any workarounds. @amandaalexandre were you able to get around this?

**Comment by amandaalexandre** [comment]
> Sorry, I wound up removing myself from the org to fix it, so I didn't investigate any workarounds. @amandaalexandre were you able to get around this?

Hi, @Scott2bReal . Not really, but now I can actually leave the organization that was giving me this problem. I'll do this and possibly this will fix my problem.

I'm curious as whether this problem was fixed for people who can't leave their organizations.
